### PR TITLE
Common Issues polish

### PR DESCRIPTION
**Pull request by anna-vasilko**
- Re-structured common issues by browser name
- Used collapsible markdown to make the doc more compact
- Removed old issues (the ones fixed and rolled out by browser vendors more than 2 years ago)
- Added some recent relevant issues, more can be added later but it should be good for now

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: COMMON_ISSUES.md [hunk 1]
```diff
@@ -1,37 +1,114 @@
...
+<p>
+  
+   It is an echo cancellation bug in Safari's implementation of WebRTC. Find more info [here](https://bugs.webkit.org/show_bug.cgi?id=213723)  
+   and [here](https://github.com/twilio/twilio-video.js/issues/1433) 
+</p>  
+</details>
+<details>
+<summary>Angular applications missing audio and/or video tracks</summary>
+<p>
+
```
Comment (manjeshbhargav) [inline COMMON_ISSUES.md:49]:
```suggestion
```

File: COMMON_ISSUES.md [hunk 2]
```diff
@@ -1,37 +1,114 @@
...
+<details>
+<summary>Angular applications missing audio and/or video tracks</summary>
+<p>
+  
+   There is a misinteraction between one of Angular's libraries, Zone.js, and
+   Safari's RTCPeerConnection APIs. For more information, see [here](https://github.com/angular/zone.js/issues/883)
+   for the issue filed against Zone.js and [here](https://bugs.webkit.org/show_bug.cgi?id=175802)
+   for the issue filed against WebKit. In order to work around this issue, you
+   should include Zone.js's webapis-rtc-peer-connection.js in your app, after
+   loading Zone.js. For example,
```
Comment (manjeshbhargav) [inline COMMON_ISSUES.md:55]:
```suggestion
   There is a mis-interaction between one of Angular's libraries, Zone.js, and
   Safari's RTCPeerConnection APIs. For more information, see [here](https://github.com/angular/zone.js/issues/883)
   for the issue filed against Zone.js, and [here](https://bugs.webkit.org/show_bug.cgi?id=175802)
   for the issue filed against WebKit. In order to work around this issue, please
   include Zone.js's webapis-rtc-peer-connection.js in your app, after
   loading Zone.js. For example,
```

File: COMMON_ISSUES.md [hunk 3]
```diff
@@ -1,37 +1,114 @@
...
+```html
+<script src="node_modules/zone.js/dist/zone.js"></script>
+<script src="node_modules/zone.js/dist/webapis-rtc-peer-connection.js"></script>
+``` 
+</p>  
+</details>
+<details>
+<summary>After unpublishing a Track, Safari 12.1 Participants cannot publish Track(s) of the same kind</summary>
+<p>
+
```
Comment (manjeshbhargav) [inline COMMON_ISSUES.md:66]:
```suggestion
```

File: COMMON_ISSUES.md [hunk 4]
```diff
@@ -1,37 +1,114 @@
...
+   Because of this Safari 12.1 [bug](https://bugs.webkit.org/show_bug.cgi?id=195489),
+   once a Participant unpublishes a MediaTrack of any kind (audio or video), it will not be able to publish another MediaTrack of the same kind.        DataTracks are not affected. We have escalated this bug to the Safari Team and are keeping track of related developments. 
+</p>  
+</details>
+
+### Safari mobile
+<details>
+<summary>Mobile Safari Participants on iOS 13.0.1 sometimes fail to send audio</summary>
+<p>
+
```
Comment (manjeshbhargav) [inline COMMON_ISSUES.md:76]:
```suggestion
```

File: COMMON_ISSUES.md [hunk 5]
```diff
@@ -1,37 +1,114 @@
...
+<summary>Mobile Safari Participants on iOS 13.0.1 sometimes fail to send audio</summary>
+<p>
+    
+   This issue happened due to regression on iOS 13.0.1. Find more details in this [WebKit bug](https://bugs.webkit.org/show_bug.cgi?id=202405). 
+</p>  
+</details>
+<details>
+<summary>Stuttering/distorted audio in Safari on iOS 14.2 and MacOS 11.0.1</summary>
+<p>
+
```
Comment (manjeshbhargav) [inline COMMON_ISSUES.md:83]:
```suggestion
```

File: COMMON_ISSUES.md [hunk 6]
```diff
@@ -1,37 +1,114 @@
...
+    
+   This issue happened due to regression in Safari's WebKit in iOS version 14.2, the fix got rolled out in iOS 14.3 beta3. Find more details [here](https://github.com/twilio/twilio-video.js/issues/1296) and in this [WebKit bug](https://bugs.webkit.org/show_bug.cgi?id=218762). 
+</p>  
+</details>
+
+### Firefox desktop
+<details>
+<summary>Firefox Participants sometimes fail to subscribe to DataTracks on Peer-to-Peer Rooms</summary>
+<p>
+
```
Comment (manjeshbhargav) [inline COMMON_ISSUES.md:92]:
```suggestion
```

File: COMMON_ISSUES.md [hunk 7]
```diff
@@ -1,37 +1,114 @@
...
+<summary>Firefox Participants sometimes fail to subscribe to DataTracks on Peer-to-Peer Rooms</summary>
+<p>
+    
+   Because of this Firefox [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1603887) Participants that join a Peer-to-Peer Room after a DataTrack      has been published by a Firefox Participant fail to subscribe to it. You can work around this issue by publishing a DataTrack while connecting to    the Room. 
+</p>  
+</details>
+<details>
+<summary>Firefox Participants cannot constrain their audio bandwidth</summary>
+<p>
+
```
Comment (manjeshbhargav) [inline COMMON_ISSUES.md:99]:
```suggestion
```

File: COMMON_ISSUES.md [hunk 8]
```diff
@@ -1,37 +1,114 @@
...
+   Participants are not able to constrain their audio bandwidth using `LocalParticipant.setParameters()`. 
+</p>  
+</details>
+
+### All browsers
+
+<details>
+<summary>Working around the browsers' autoplay policy (which might block audio and video from playing)</summary>
+<p>
+
```
Comment (manjeshbhargav) [inline COMMON_ISSUES.md:110]:
```suggestion
```

File: COMMON_ISSUES.md [hunk 9]
```diff
@@ -96,177 +173,31 @@ For RemoteVideoTracks, there are two ways to ensure playback:
...
-Aggressive Browser Extensions and Plugins
------------------------------------------
-
-Some browser extensions and plugins will disable WebRTC APIs, causing
+</p>  
+</details>
+<details>
+<summary>Support of Beta/Canary/Nightly browser releases</summary>
+<p>
+
```
Comment (manjeshbhargav) [inline COMMON_ISSUES.md:181]:
```suggestion
```

File: COMMON_ISSUES.md [hunk 10]
```diff
@@ -96,177 +173,31 @@ For RemoteVideoTracks, there are two ways to ensure playback:
...
+    
+   We constantly test for and ensure compatibility with the current stable and beta releases of supported browsers. 
+   However, because some of the APIs we rely upon, like WebRTC, are under active development in the browsers, 
+   we cannot guarantee compatibility with Canary or Nightly releases. Find our browsers support matrix [here](https://www.twilio.com/docs/video/javascript#supported-browsers)
+</p>  
+</details>
+<details>
+<summary>Internet Explorer and other WebRTC-incompatible Browsers</summary>
+<p>
+
```
Comment (manjeshbhargav) [inline COMMON_ISSUES.md:190]:
```suggestion
```

File: COMMON_ISSUES.md [hunk 11]
```diff
@@ -96,177 +173,31 @@ For RemoteVideoTracks, there are two ways to ensure playback:
...
+    
+   twilio-video.js requires WebRTC, which is not supported by Internet Explorer.
+   While twilio-video.js will load in Internet Explorer and other browsers that do not support WebRTC, 
+   attempting to connect to a Room or attempting to acquire LocalTracks will fail. Find our browsers support matrix [here](https://www.twilio.com/docs/video/javascript#supported-browsers)
+</p>  
+</details>
+<details>
+<summary>Aggressive Browser Extensions and Plugins</summary>
+<p>
+
```
Comment (manjeshbhargav) [inline COMMON_ISSUES.md:199]:
```suggestion
```

File: COMMON_ISSUES.md [hunk 12]
```diff
@@ -1,37 +1,114 @@
...
+Please review this list of known issues and workarounds before opening a new ticket.  
+We recommend regularly upgrading to the latest version of the SDK, which includes bug fixes and improvements, see 
+[CHANGELOG.md](CHANGELOG.md).
+
+
+### Chrome desktop
+<details>
+<summary>Chrome memory leak might cause degraded experience in group rooms</summary>
+<p>
+
```
[the remark this answers is not in the document]
Comment (anna-vasilko) [reply COMMON_ISSUES.md:14]: @manjeshbhargav In collapsable markdown it seems to be required to have this extra line in order to preserve some of the styling in subsequent text. For example, with removal of this line, hyperlinks won't work in subsequent text.  That is the reason it is present in each of the collapsable items.

**Comment by anna-vasilko** [comment]
@manjeshbhargav I have included all of your wording suggestions. Thanks! Some of the extra lines, I am afraid, have to stay to preserve the collapsable markdown styling, otherwise hyperlinks in them won't work.

File: COMMON_ISSUES.md [hunk 13]
```diff
@@ -39,7 +37,7 @@ We recommend regularly upgrading to the latest version of the SDK, which include
 <summary>Echo issues in Safari when using external microphone</summary>
 <p>
   
-   It is an echo cancellation bug in Safari's implementation of WebRTC. Find more info [here](https://bugs.webkit.org/show_bug.cgi?id=213723)  
+   This is an echo cancellation bug in Safari's implementation of WebRTC. For more details, go [here](https://bugs.webkit.org/show_bug.cgi?id=213723).
    and [here](https://github.com/twilio/twilio-video.js/issues/1433)
```
Comment (manjeshbhargav) [inline COMMON_ISSUES.md:41]:
```suggestion
   and [here](https://github.com/twilio/twilio-video.js/issues/1433).
```

**Review by manjeshbhargav — APPROVED** [review]

**Pull request merged**